akkeeper-yarn/src/main/scala/akkeeper/yarn/YarnLocalResourceManager.scala

```diff
@@ -55,11 +58,19 @@ private[akkeeper] final class YarnLocalResourceManager(conf: Configuration,
     dst
   }
 
+  def getLocalResource(
+    dstPath: Path,
+    localResourceType: LocalResourceType = LocalResourceType.FILE,
+    localResourceVisibility: LocalResourceVisibility = LocalResourceVisibility.APPLICATION
+  ): LocalResource = {
+    val dstFs = dstPath.getFileSystem(conf)
+    val dstStatus = dstFs.getFileStatus(dstPath)
+    create(dstFs, dstStatus, localResourceType, localResourceVisibility)
+  }
+
   def createLocalResource(srcStream: InputStream, dstPath: String): LocalResource = {
-    val dstFs = stagingDirPath.getFileSystem(conf)
-    val dst = copyResourceToStagingDir(dstFs, srcStream, dstPath)
-    val dstStatus = dstFs.getFileStatus(dst)
-    create(dstFs, dstStatus)
+    val dst = copyResourceToStagingDir(srcStream, dstPath)
```

Comment:
[Thought|Optional] Perhaps it makes sense to rename all `createLocalResource` methods to `uploadLocalResource` to draw a clear line between the case when we copy the resource and when we use the original source instead.

Reply:
Renamed